Kinvey/KinveyTests/AutoDataStoreSpec.swift

```diff
@@ -1642,7 +1645,7 @@ class AutoDataStoreSpec: QuickSpec {
 
 }
 
-func it(_ description: String, flags: FilterFlags = [:], file: String = #file, line: UInt = #line, closure: @escaping () throws -> Void) {
+func it(_ description: String, flags: FilterFlags = [:], file: FileString = #file, line: UInt = #line, closure: @escaping () throws -> Void) {
```

Comment:
Why is the change made to use `FileString` here instead of the standard `String`?

Reply:
that's because Swift 5.1 now have a special type for Strings that are generated by the compiler such as `#file`